test/functional/feature_fee_estimation.py

```diff
@@ -131,18 +131,17 @@ def check_estimates(node, fees_seen):
 
 def send_tx(node, utxo, feerate):
     """Broadcast a 1in-1out transaction with a specific input and feerate (sat/vb)."""
-    overhead, op, scriptsig, nseq, value, spk = 10, 36, 5, 4, 8, 24
-    tx_size = overhead + op + scriptsig + nseq + value + spk
-    fee = tx_size * feerate
-
     tx = CTransaction()
     tx.vin = [
         CTxIn(COutPoint(int(utxo["txid"], 16), utxo["vout"]), REDEEM_SCRIPT)
     ]
-    tx.vout = [CTxOut(int(utxo["amount"] * COIN) - fee, P2SH)]
-    txid = node.sendrawtransaction(tx.serialize().hex())
+    tx.vout = [CTxOut(int(utxo["amount"] * COIN), P2SH)]
+
+    # vbytes == bytes as we are using legacy transactions
+    fee = len(tx.serialize()) * feerate
```

Comment:
in 7ccd775dcc5f363c9f35409e9170b1e07bfdc753:

Why not just use `tx.get_vsize()` ?